BraveShared/BraveStrings.swift

```diff
@@ -428,6 +428,9 @@ public extension Strings {
     public static let Copy_app_info_to_clipboard = NSLocalizedString("CopyAppInfoToClipboard", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Copy app info to clipboard.", comment: "Copy app info to clipboard action sheet action.")
     public static let Block_3rd_party_cookies = NSLocalizedString("Block3rdPartyCookies", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Block 3rd party cookies", comment: "cookie settings option")
     public static let Block_all_cookies = NSLocalizedString("BlockAllCookies", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Block all cookies", comment: "cookie settings option")
+    public static let Block_all_cookies_alert_info = NSLocalizedString("BlockAllCookiesAlertInfo", tableName: "BraveShared", bundle: Bundle.braveShared, value: "This Setting does this blah blah blah", comment: "Information displayed to user when block all cookie is enabled.")
+    public static let Block_all_cookies_alert_title = NSLocalizedString("BlockAllCookiesAlertTitle", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Block all cookies alert title", comment: "Title of alert displayed to user when block all cookie is enabled.")
+    public static let Block_all_cookies_failed_alert_msg = NSLocalizedString("BlockAllCookiesFailedAlertMsg", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Block all cookies failed alert message", comment: "Message of alert displayed to user when block all cookie operation fails")
```

Comment:
Need strings for these!

Reply:
We should probably get these from product before merging

Reply:
https://github.com/brave/brave-ios/issues/686